.eslintrc

```diff
@@ -16,7 +16,7 @@
 	"env": {
 		"browser": true
 	},
-	"ignorePatterns": ["**/lang/**/*.json", "**/*.d.ts"],
+	"ignorePatterns": ["rollup.config.mjs", "**/lang/**/*.json", "**/*.d.ts"],
```

Comment:
rollup config is removed from linting? any particular reason?

Reply:
Sorry that I forgot how to contain everything in a single review for a couple of these!